conftest.py

```diff
@@ -21,6 +21,11 @@
     MPI = None
 
 
+def pytest_collectstart(collector):
+    if collector.fspath and collector.fspath.ext == '.ipynb':
+        collector.skip_compare += ('text/latex',)
```

Comment:
see https://nbval.readthedocs.io/en/latest/#Skipping-certain-output-types

This just avoid failures for different sympy version printing slightly different latex

Reply:
Also needed to be moved to root directory so that `conftest` is found when testing the examples

Reply:
Good to know, so this cancels out my LATEX changes in the other PR?